charts/actions-runner-controller/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.12.7
+version: 0.12.8
```

Comment:
I'll revert this version change before merging, so that we won't publish a chart that requires unreleased version of the controller :)
I think we'll be releasing the new version of the controller in a few weeks so once it is released I'll bump the chart version

Reply:
Ack, I bumped it due to the CDR changes.